components/MintPage/MintDay/Modals/MintMoreModal.tsx

```diff
@@ -102,10 +102,10 @@ const MintMoreModal: FC<MintMoreModalProps> = ({ isModalVisible, toggleIsVisible
               // eslint-disable-next-line react/jsx-no-useless-fragment
               <></>
             )}
-            {lockedCntOfCre8or ? (
-              <MintModalCTAButton link="/profile" id="profile_link_btn" label="set up profile" />
+            {!lockedCntOfCre8or ? (
```

Comment:
I thought of this flow.
if lockckedCntOfCre8or === 18 - leftQuantityCount => set up profile
else enter ware house 

What is my mistake or can below two codes reach the same result?
!lockecdCntOfcCre8or => lockedCntOfCre8or = 0
In my case, lockckedCntOfCre8or !== 18 - leftQuantityCount